independent-publisher-connectors/Google Photos/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[YOUR_CLIENT_ID]",
+          "scopes": [
+            "https://www.googleapis.com/auth/photoslibrary https://www.googleapis.com/auth/photoslibrary.edit.appcreateddata https://www.googleapis.com/auth/photoslibrary.sharing https://www.googleapis.com/auth/photoslibrary.edit.appcreateddata"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://accounts.google.com/o/oauth2/v2/auth"
+            },
+            "tokenUrl": {
+              "value": "https://accounts.google.com/o/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://accounts.google.com/o/oauth2/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Julia Muiruri"

```

Comment:
@juliamuiruri4 please add the "stackOwner" property, you can follow this template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).
Please add "stackOwner": "Name of company or organization that connector is connecting to."